calicoctl/commands/get.go

```diff
@@ -169,7 +165,15 @@ Description:
 		os.Exit(1)
 	}
 
-	results := executeConfigCommand(parsedArgs, actionList)
+	results := commandResults{}
+
+	name := argutils.ArgStringOrBlank(parsedArgs, "<NAME>")
+	if name == "" {
+		results = executeConfigCommand(parsedArgs, actionList)
```

Comment:
I don't think this is the correct fix since it won't correctly handle file-based input.

And I assume with file-based input you'll also need to worry about whether each entry specifies a name/namespace and use get/list accordingly.

Probably what I would do is have the ResourceManager expose a Get() interface which actually maps to either the List or Get call depending on whether the namespace or name is wild carded.